server/modules/collectImport/service/collectImport/dataImportJobs/recordsImportJob.js

```diff
@@ -35,7 +35,9 @@ export default class RecordsImportJob extends Job {
 
   async onStart () {
     await super.onStart()
-    await RecordManager.disableTriggers(this.surveyId, this.tx)
+
+    // Speed up processing a bit:
+    await this.tx.query('SET CONSTRAINTS ALL DEFERRED')
```

Comment:
I don't know if it's better to wrap this somehow with the DbUtils class...

Reply:
I wouldn't normally wrap small one-off things. I think there's such a thing as premature generalization. One rule of thumb is to start doing that once there's three or more usages of something.